### PR TITLE
Allow to set some meeting values in meeting.clone.

### PR DESCRIPTION
**Pull request by reiterl**
Add a test.

### CODE REVIEW

File: tests/system/action/meeting/test_clone.py [hunk 1]
```diff
@@ -118,6 +123,30 @@ def test_clone_with_users(self) -> None:
         self.assert_model_exists("meeting/1", {"user_ids": [1]})
         self.assert_model_exists("meeting/2", {"user_ids": [1]})
 
+    def test_clone_with_set_fields(self) -> None:
+        self.set_models(self.test_models)
+
+        response = self.request(
+            "meeting.clone",
```
Comment (r-peschke) [inline tests/system/action/meeting/test_clone.py:130]: Please check also for start_time and end_time

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: tests/system/action/meeting/test_clone.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/meeting/test_clone.py:130]: Add a test for start_time and end_time.

File: openslides_backend/action/actions/meeting/clone.py [hunk 2]
```diff
@@ -51,6 +59,19 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
...
             self.get_meeting_from_json(meeting_json)["committee_id"] = committee_id
 
+        # pre update the meeting
+        for field in (
+            "welcome_title",
+            "description",
+            "start_time",
+            "end_time",
+            "location",
+            "organization_tag_ids",
```
Comment (jsangmeister) [inline openslides_backend/action/actions/meeting/clone.py:69]: extract into array so you can deduplicate with the payload above, e.g. `updatable_fields`

**Review by jsangmeister — CHANGES_REQUESTED** [review]
minor change

File: openslides_backend/action/actions/meeting/clone.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/clone.py]: Extract the fields into "updatable_fields".

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**